lib/foreman_inventory_upload/async/generate_all_reports_job.rb

```diff
@@ -13,7 +13,13 @@ def perform
         organizations = Organization.unscoped.all
 
         organizations.map do |organization|
```

Comment:
This previously returned an array of active jobs and now returns an array where elements can be either active jobs or `nil`s. Is the result used anywhere? If not, could we switch `#map` for `#each`?

Reply:
The method anyway was returning the `ensure` block. I think the `map` could be replaced with `each` anyway.

Reply:
I always thought `ensure` blocks get evaluated last, but do not alter the return value

Reply:
Yep, you're right. `ensure` does not return value.

Well, removed `nil` values. Now the return value will filter out `nil` values.